### PR TITLE
add conlleval.pl

### PR DESCRIPTION
**Pull request by slefrancois**
The URL for the file conlleval.pl no longer works. This adds the file to the repo, with the custom prints we added for processing the output.

### CODE REVIEW

File: code/conlleval.pl [hunk 1]
```diff
@@ -0,0 +1,320 @@
+#!/usr/bin/perl -w
+# conlleval: evaluate result of processing CoNLL-2000 shared task
+# usage:     conlleval [-l] [-r] [-d delimiterTag] [-o oTag] < file
+#            README: http://www.clips.uantwerpen.be/conll2000/chunking/output.html
+# options:   l: generate LaTeX output for tables like in
+#               http://cnts.uia.ac.be/conll2003/ner/example.tex
```
Comment (lamblin) [inline code/conlleval.pl]: Maybe over-eager, but that URL does not work and seems to have been replaced by http://www.clips.uantwerpen.be/conll2003/ner/example.tex

File: code/conlleval.pl [hunk 2]
```diff
@@ -0,0 +1,320 @@
...
+#            o: alternative outside tag (default is O)
+# note:      the file should contain lines with items separated
+#            by $delimiter characters (default space). The final
+#            two items should contain the correct tag and the 
+#            guessed tag in that order. Sentences should be
+#            separated from each other by empty lines or lines
+#            with $boundary fields (default -X-).
+# url:       http://www.clips.uantwerpen.be/conll2000/chunking/
+# started:   1998-09-25
+# version:   2004-01-26
```
Comment (lamblin) [inline code/conlleval.pl]: Should we update the version number then?

File: code/conlleval.pl [hunk 3]
```diff
@@ -16,7 +16,7 @@
 #            with $boundary fields (default -X-).
 # url:       http://www.clips.uantwerpen.be/conll2000/chunking/
 # started:   1998-09-25
-# version:   2004-01-26
+# version:   2018-03-09
 # author:    Erik Tjong Kim Sang <erikt@uia.ua.ac.be>
```
Comment (nouiz) [inline code/conlleval.pl]: Maybe change author to "original author" to have this more clear. Also indent the "with modification" under "Erik" to have the modification more clear

**Comment by nouiz** [comment]
As @lamblin already approuved, I'll merge.

**Pull request merged**